src/PostCampaign.php

```diff
@@ -531,7 +533,6 @@ public static function replace_font_aliases( array $css_vars ): array {
 	 */
 	private static function migrate_old_vars( array $css_vars ): array {
 		$mappings = [
-			'footer_color'            => [ 'footer--background' ],
```

Comment:
Whereas the other ones are saved on the post, this one is only set by the preceding code. So instead I made it use the right one in the first place.